chingu-voyage3/geckos-02
# Carousel

## Purpose
* Create a carousel of hero images for homepage

## Approach/Fixes
* Created carousel folder as a component of `src`
* Used @thorbw 's branch as a reference for adding SASS
* This PR is to merge it in to the 'skeleton' branch (which may as well be a development branch)

## Testing
* Download this PR to your local machine
* Run `npm start`
* Check that the carousel shows and you can navigate through the different slides
* Resize the window to check for mobile response

## Notes
* The carousel doesn't have any animation yet (just hides and shows next slide; thinking of using `ReactCSSTransitionGroup` for a slide or fade effect)
* Temporarily edited the navigation at the top with styles (subject to change)
* All styles are just temporary so we can get a feel; anything can be changed to have consistent styling for the future
* Currently storing the data for each slide in a separate js file (this might not be best practice so if you have better recommendations please let me know)
* The `join now` button is currently empty
* The `contact` button will later anchor to the footer with contact/social media information
* When I installed npm, it created a package-lock.json file (dunno if it's super important that we delete it or not, but I kept it for now)
* My JS code has a lot of linting errors as of now (I plan on refactoring it soon)
* Made html and body padding and margin set to 0 in the carousel css file (to be replace with a global css file)
